pkg/network/setup/netstat_test.go

```diff
@@ -512,6 +516,40 @@ var _ = Describe("netstat", func() {
 
 			Expect(setup.Vmi.Status.Interfaces).To(BeEmpty())
 		})
+
+		DescribeTable("verify primary interface is always first in Status.Interfaces list", func(vmiSpecIfaces []v1.Interface, domainIfaces []api.Interface, statusIfaces []v1.VirtualMachineInstanceNetworkInterface) {
```

Comment:
Why are the tests under `misc scenario`. I think the purpose of the `misc scenario` is to contain "strange" scenarios that shouldn't happen on a real system.

Reply:
done